e2e-tests/commands/shared-commands/assertions/assert-loss-payee-financial-international-field-values.js

```diff
@@ -0,0 +1,34 @@
+import { field as fieldSelector, autoCompleteField } from '../../../pages/shared';
+import { POLICY as POLICY_FIELD_IDS } from '../../../constants/field-ids/insurance/policy';
+import mockApplication from '../../../fixtures/application';
+
+const { LOSS_PAYEE_FINANCIAL_INTERNATIONAL } = mockApplication.POLICY;
+
+const {
+  LOSS_PAYEE_FINANCIAL_INTERNATIONAL: {
+    BIC_SWIFT_CODE, IBAN,
+  },
+  FINANCIAL_ADDRESS,
+} = POLICY_FIELD_IDS;
+
+/**
+ * assertLossPayeeFinancialInternationalFieldValues
+ * Assert all field values in the "loss payee financial international" form.
+ * @param {String} expectedAccountNumber: Account number
+ * @param {String} expectedSortCode: Sort code
+ * @param {String} expectedFinancialAddress: Financial address
+ */
+const assertLossPayeeFinancialInternationalFieldValues = ({
+  expectedIban = LOSS_PAYEE_FINANCIAL_INTERNATIONAL[IBAN],
+  expectedBicSwiftCode = LOSS_PAYEE_FINANCIAL_INTERNATIONAL[BIC_SWIFT_CODE],
+  expectedFinancialAddress = LOSS_PAYEE_FINANCIAL_INTERNATIONAL[FINANCIAL_ADDRESS],
```

Comment:
Can we change this to be `EXPECTED_MULTI_LINE_STRING` please, to ensure it works for multi line strings (otherwise will probably have a bug). Same for UK fields?

There's an example in `cy.assertAgentDetailsFieldValues`.

If you add this in I suspect it will currently fail, probably need to update

`mapApplicationToFormFields > mapTextareaFields`

And add test coverage in

```js
submit-policy-textarea-fields-with-new-line-characters.spec.js
submit-policy-textarea-fields-with-special-characters.spec.js
```

Maybe should be a new PR

Reply:
it works without failing

Reply:
If `mapApplicationToFormFields > mapTextareaFields` has _not_ been updated, and we have the above E2E tests, they will fail.

This will get picked up by QA, won't take long to do 😊

Reply:
done